File: bronze/5054/c++/main.cpp
#include <iostream>

int main()
{
    int num, rep, min, max, temp;
    std::cin >> num;
    for (int i = 0; i < num; i++) {
        std::cin >> rep;
        min = 99; max = 0;
        for (int j = 0; j < rep; j++) {
            std::cin >> temp;
            if (min > temp) { min = temp; }
            if (max < temp) { max = temp; }
        }
        std::cout << ((max - min) * 2) << '\n';
    }

    return 0;
}

File: bronze/5054/python/main.py
  
if (__name__ == "__main__"):
    num = int(input())
    for _ in range(num):
        rep = int(input())
        temp = list(map(int, input().split()))
        min_, max_ = min(temp), max(temp)
        print((max_ - min_) *2)